tensorflow_datasets/core/features/image_feature_test.py

```diff
@@ -52,7 +52,7 @@ def test_images(self, dtype):
       img_file_expected_content *= 257  # Scale int16 images
 
     self.assertFeature(
-        feature=features_lib.Image(dtype=dtype),
+        feature=features_lib.Image(dtype=dtype, encoding_format='png'),
```

Comment:
We want to test paths and file_obj with `encoding_format=None`, but np.array with `encoding_format='pnng'`